drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -569,7 +569,11 @@ static void set_up_plls(void)
 		stm32_clock_switch_to_hsi();
 		LL_RCC_SetAHBPrescaler(ahb_prescaler(1));
 	}
+    /* Disable PLL */
 	LL_RCC_PLL_Disable();
+    while (LL_RCC_PLL_IsReady() != 0U) {
```

Comment:
Please fix indentation (we're using 8 char tabs)

Reply:
Fixed. Also added the LL_RCC_PLL2_IsReady() check as @FRASTM mentioned.